.circleci/config.yml

```diff
@@ -38,6 +38,7 @@ jobs:
                             --v8 "$V8" \
                             --npm "$NPM" \
                             --tag "$TAG" \
+                            --latest "$LATEST" \
```

Comment:
from where is picked `$LATEST`?

Reply:
LATEST will be provided by https://github.com/nearform/nodejs-release-watcher
It will mark the latest version from https://nodejs.org/dist/index.json to be the LATEST.